src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -547,7 +547,7 @@ public sealed partial class JsonArray : System.Text.Json.Nodes.JsonNode, System.
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         public override void WriteTo(System.Text.Json.Utf8JsonWriter writer, System.Text.Json.JsonSerializerOptions? options = null) { }
     }
-    public abstract partial class JsonNode : System.Dynamic.IDynamicMetaObjectProvider
+    public abstract partial class JsonNode
```

Comment:
Can we also remove the System.Linq.Expressions references in the ref .csproj?

Reply:
Yes. Thanks